jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesViewImpl.java

```diff
@@ -65,87 +70,113 @@
     @DataField("pagination")
     private PaginationViewImpl pagination;
 
-    List allElementsList = new ArrayList();
-
-    private Command userAddCommand;
+    List <CaseRoleItemView> allFilteredElements = new ArrayList();
```

Comment:
These collections seems like state, ideally this should only be kept in the presenter. Same goes for actual filtering.